packages/runtime-dom/src/helpers/useCssVars.ts

```diff
@@ -80,7 +80,9 @@ function setVarsOnNode(el: Node, vars: Record<string, string>) {
   if (el.nodeType === 1) {
     const style = (el as HTMLElement).style
     for (const key in vars) {
-      style.setProperty(`--${key}`, vars[key])
+      if (vars[key] !== undefined && vars[key] !== null) {
```

Comment:
```suggestion
      if (vars[key] != null) {
```

Reply:
Hi @Justineo, could you elaborate more on this RC?
I don't think there's any need placing `undefined` on a CSS property.

Reply:
Actually, using the strict version of inequality operator is a common practice and a good idea. As a side affect, I just got confused by that too, it struct later on.
To increase readability, I'd like to suggest using the strict operator comparison

Reply:
This is a common special case for the strict equal rule to perform nullish checks.

Reply:
Yeah, it does. But it's not clear at first glance or even doesn't get understood if you don't know how it works, that's what I'm saying 😄 It creates a confusion and it should be avoided
There's even an [ESLint rule](https://eslint.org/docs/latest/rules/eqeqeq) that makes you forget about non-strict equality operators

Reply:
> There's even an [ESLint rule](https://eslint.org/docs/latest/rules/eqeqeq) that makes you forget about non-strict equality operators

By "common special case" I mean there is an option for the `eqeqeq` rule that allows non-strict equality checks for `null`. And we are already using non-strict equality checks for `null` in other places in the Vue codebase so I think we should keep them consistent.